File: datasets/IR-Plag/convertai-original-java-rest-python/case-02/non-plagiarized/05/T02.py
import math

radius_length = input("Enter the radius and length of a cylinder: ")
radius, length = map(float, radius_length.split())

area = round(radius * radius * math.pi, 4)
volume = round(radius * radius * math.pi * length, 1)

print(f"The area is {area}")
print(f"The volume is {volume}")

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-02/non-plagiarized/05/T02.ts
import * as readline from 'readline';

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Enter the radius and length of a cylinder: ', (radlen) => {
  const splited = radlen.split(/\s+/);
  const rad = parseFloat(splited[0]);
  const len = parseFloat(splited[1]);

  console.log(`The area is ${Math.round(rad * rad * 3.14159 * 10000.0) / 10000.0}`);
  console.log(`The volume is ${Math.round(rad * rad * 3.14159 * len * 10.0) / 10.0}`);

  rl.close();
});

